apps/explorer/src/ui/utils/ButtonOrLink.tsx

```diff
@@ -1,21 +1,37 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { type ComponentProps, forwardRef } from 'react';
+import { type ComponentProps, forwardRef, useMemo } from 'react';
 
 import { LinkWithQuery, type LinkProps } from './LinkWithQuery';
 
 export interface ButtonOrLinkProps
     extends Omit<
         Partial<LinkProps> & ComponentProps<'a'> & ComponentProps<'button'>,
         'ref'
-    > {}
+    > {
+    prefixIcon?: JSX.Element;
+    postfixIcon?: JSX.Element;
+}
 
 export const ButtonOrLink = forwardRef<
```

Comment:
This feels more like a concern for our button component and link components. This component should just pass children directly.

Reply:
@Jordan-Mysten  If we pass through children, then the changes will need to be duplicated in both the `Button` and `Link` Components, wrapping the `children` there rather than being inside `ButtonOrLink`. Are you okay with doing that?

Reply:
Yep!

Reply:
updated